### DISCUSSION TITLE
problem in following the nextjs official website's introduction of nextjs

### DISCUSSION THREAD

**Discussion by fuliez** [vercel/next.js#65370] [Help]
### Summary

I want to follow the website https://www.nextjs.cn/learn/basics/create-nextjs-app/setup to learn nextjs.
But when I try to run `npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter" ` to creat a nextjs app as the web suggests,I met the problem as shown in the picture.I think it suggests that the repository is missing.
How can I get started to follow the web course.Or is there another new course?Thank you.
![image](https://github.com/vercel/next.js/assets/108499205/94b87106-2802-4062-991f-72074268fdb3)

**Comment by SouthLink** [comment]
This is not the official document, some examples of the warehouse may have failed, you should learn through https://nextjs.org/docs, the official document, if you need a Chinese document, I recommend this https://qufei1993.github.io/nextjs-learn-cn/

**Comment by weijunext** [comment]
Check out the new and better Chinese Docs: https://github.com/vercel/next.js/discussions/71980

**Comment by aniruddhaadak80** [comment]
Hi @fuliez ,

It seems you're encountering an issue with the Next.js setup while following the official website's guide. The error may indicate that the specified example repository is missing or unavailable. Here are a few steps you can try to get started:

1. **Check Repository URL**: Verify that the URL you are using is correct. The example may have been moved or renamed. You can check the [official Next.js examples repository](https://github.com/vercel/next.js/tree/canary/examples) for updated examples.

2. **Use the Default Template**: Instead of the specified example, you can create a Next.js app with the default template by running:
   ```bash
   npx create-next-app nextjs-blog --use-npm
   ```

3. **Explore New Courses**: If you're looking for other learning resources, the official Next.js documentation has a section on [getting started](https://nextjs.org/docs/getting-started) which might be helpful.

Thanks!
